src/main/java/com/twitter/http2/DefaultHttp2Headers.java

```diff
@@ -0,0 +1,65 @@
+package com.twitter.http2;
+
+import io.netty.handler.codec.DefaultHeaders.NameValidator;
+import io.netty.handler.codec.http.DefaultHttpHeaders;
+import io.netty.util.AsciiString;
+import io.netty.util.ByteProcessor;
+import io.netty.util.internal.PlatformDependent;
+
+import static io.netty.util.internal.ObjectUtil.checkNotNull;
+
+public class DefaultHttp2Headers extends DefaultHttpHeaders {
+  private static final ByteProcessor HEADER_NAME_VALIDATOR = new ByteProcessor() {
+    @Override
+    public boolean process(byte value) throws Exception {
+      validateChar((char) (value & 0xFF));
+      return true;
+    }
+  };
+
+  private static void validateChar(char character) {
+    switch (character) {
+      case '\t':
+      case '\n':
+      case 0x0b:
+      case '\f':
+      case '\r':
+      case ' ':
+      case ',':
+      case ';':
+      case '=':
+        throw new IllegalArgumentException(
+                "a header name cannot contain the following prohibited characters: =,; \\t\\r\\n\\v\\f: " +
```

Comment:
Good catch, the exception in netty 4.1 lists it twice

    codec-http/src/main/java/io/netty/handler/codec/http/DefaultHttpHeaders.java 
    "a header name cannot contain the following prohibited characters: =,;: \\t\\r\\n\\v\\f: " +